src/renderer/global.d.ts

```diff
@@ -29,7 +29,6 @@ declare global {
       | undefined
     __refetchChatlist: undefined | (() => void)
     __welcome_qr: undefined | string
```

Comment:
makes sense to remove `__welcome_qr` here too if you have moved it already to the new instant onboarding context